spec/factories/allocation.rb

```diff
@@ -0,0 +1,37 @@
+require 'faker'
```

Comment:
Is this meant to be in this PR?

Reply:
The Faker gem is for creating dummy data. Once the migration is run this factory will be deleted

Reply:
Yep (it's for creating dummy data, however this factory will be removed once the migration is run)